src/pages/zoeken/[slug]/[ie]/index.tsx

```diff
@@ -554,13 +553,16 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, url }) => {
 		playableUrl: string | undefined,
 		representation: IeObjectRepresentation | undefined
 	): ReactNode => {
+		if (!playableUrl && !mediaInfo) {
+			return <></>;
```

Comment:
Null is also a reactNode ;) 
```suggestion
			return null;
```